src/network/protocol/grandpa_warp_sync.rs

```diff
@@ -49,7 +49,6 @@ pub fn decode_grandpa_warp_sync_response(
                                 ))
                             })
                     },
-                    crate::util::nom_scale_compact_usize,
```

Comment:
Do we want to request that this value is kept in the response?

Reply:
I said in DM that think it made more sense to keep it. However, after thinking twice about it, I think that this length prefix indeed doesn't really make sense. After all, we can't do much with the response is we can't know the format of a justification.
Plus, this protocol is pretty tied to the Grandpa algorithm. If we implemented another finality algorithm in Substrate, it would likely have to use a different protocol. Therefore we don't need to take into account the fact that justifications might in fact not be Grandpa justifications.